selenium_tests/class_roster/class_roster_permission_tests.py

```diff
@@ -35,7 +35,7 @@ def test_roles_access(self, user_id, given_access):
                 self.course_admin_dashboard_page
                     .class_roster_button_is_displayed(),
                 'User {} should see the class roster button on page but '
```

Comment:
Great catch.  Somehow I missed this - thanks. Will have it fixed for the new PR.